example/react/screens/PlaylistScreen.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import TrackPlayer, { ProgressComponent } from 'react-native-track-player';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+
+import Player from '../components/Player';
+import playlistData from '../data/playlist.json';
+
+import PlayerStore from '../stores/Player';
+import TrackStore, { playbackStates } from '../stores/Track';
+
+@observer
+export default class LandingScreen extends Component {
+  static navigationOptions = {
+    title: 'Playlist Example',
+  };
+
+  componentDidMount() {
+    TrackPlayer.setupPlayer();
```

Comment:
You should retrieve the player state or enable the [stopWithApp](https://github.com/react-native-kit/react-native-track-player/wiki/Documentation#updateoptionsdata) option